src/vm/eetwain.cpp

```diff
@@ -5729,6 +5729,7 @@ BOOL EECodeManager::IsInFilter(GCInfoToken gcInfoToken,
 }
 
 
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
```

Comment:
These methods are specific to !WIN64EXCEPTIONS exception handling scheme. So I would just make the ifdef consistent with header file here.

In the unlikely case somebody tries to build say `<some new platform> + !WIN64EXCEPTIONS` combination, they will get a ton of build breaks. I do not think we need to worry about it.